.github/workflows/android-cd.yml

```
@@ -0,0 +1,109 @@
+                run: |
+                    ./gradlew :app:bundleRelease
+
+            -   name: Upload Release Build to Artifacts
+                uses: actions/upload-artifact@v4
+                with:
+                    name: release-artifacts
+                    path: app/build/outputs/bundle/release/
+                    if-no-files-found: error
+
+            -   name: Create Github Release
+                uses: softprops/action-gh-release@v1
```

actionlint (1.7.7)

the runner of "softprops/action-gh-release@v1" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)